ngraph/core/include/ngraph/pass/graph_rewrite.hpp

```diff
@@ -219,11 +219,29 @@ namespace ngraph
             void set_pass_config(const std::shared_ptr<PassConfig>& pass_config) override;
 
         protected:
+            bool apply_matcher_passes(std::shared_ptr<Function> f,
+                                      std::deque<std::shared_ptr<Node>> nodes_to_run);
```

Comment:
shouldn't nodes_to_run be an rvalue reference?

Reply:
I think there is no need since we won't make a copy here.